### DISCUSSION TITLE
Multiple chrome running simultaneously question about use_subprocess

### DISCUSSION THREAD

**Discussion by zmttmz** [ultrafunkamsterdam/undetected-chromedriver#1660] [Q&A] [answered]
Hello,
I have some doubts about how to use the "use_subprocess" in the following situation:

I have a phyton script which priorly opens 15 chrome browsers with 15 chromdrivers simultanteously, and keep these opened all the time. Then i am sending requests to these browsers (checking first which one is free - and then using the free one, if none free i pass the request and try later). My question is, in this case which is best to use? **use_subprocess=False** or **use_subprocess=True**

Thank you,
Bogdan

**Comment by ultrafunkamsterdam** [comment, marked as answer]
Just leave it the default

**Reply by zmttmz** [reply]
thanks
